src/drizzle/schema.ts

```diff
@@ -1066,3 +1066,13 @@ export const itemTagsTable = pgTable(
     unique('UQ_item_tag').on(table.tagId, table.itemId),
   ],
 );
+
+export const maintenanceTable = pgTable(
+  'maintenance',
+  {
```

Comment:
Should this table have a primary key even if we do not really need it ?
Something like a generated always identity https://orm.drizzle.team/docs/latest-releases/drizzle-orm-v0320#-postgresql-identity-columns ?

Reply:
No worries, I read it in the docs, but I think it is equivalent. You have to use the table version If you want to devine a unique constraint on multiple columns. But for single columns I feel using it inline on the column def is easier.